src/main/java/controller/DeleteProductServlet.java

```diff
@@ -0,0 +1,30 @@
+package controller;
+
+import factory.ProductServiceFactory;
+import service.ProductService;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+@WebServlet(value = "/products/delete")
+public class DeleteProductServlet extends HttpServlet {
+
+    private static final ProductService productService = ProductServiceFactory.getProductService();
+
+    @Override
+    protected void doGet(HttpServletRequest request,
+                         HttpServletResponse response) throws ServletException, IOException {
+        String id = request.getParameter("id");
+        final HttpSession session = request.getSession();
```

Comment:
do we need 23 and 24 line?
and you remove response.sendRedirect();

Reply:
if id==null? what happens? maybe better write the message to user about it?
the same in class DeleteUserServlet